Client/src/Service/Mixins/SearchReportsService.ts

```diff
@@ -6,8 +6,10 @@ import {
   SearchConfig,
   Answer,
   FilterValueArray,
+  Identifier,

```

Comment:
Do you need this import?

Reply:
Nope! I will remove it. Thanks for spotting it!